core-bundle/src/Resources/contao/library/Contao/Widget.php

```diff
@@ -982,6 +982,10 @@ protected function validator($varInput)
 
 				case 'url':
 					$varInput = StringUtil::specialcharsUrl($varInput);
+					
+					if ($this->decodeEntities) {
```

Comment:
```suggestion
					if ($this->decodeEntities)
					{
```
Legacy coding style ☺️